Cue/app/actions/login.js

```diff
@@ -26,35 +29,32 @@ function logOut(): Action {
 }
 
 
-
-async function loadUsername(name): PromiseAction {
+async function loadUsername(name: string): PromiseAction {
+  UserApi.setName(name).catch(e => console.warn('Could not set name on Cue server'))
   return {
     type: 'LOADED_USERNAME',
     name,
   };
 }
 
+
 function serverLogin(): ThunkAction {
   return (dispatch, getState) => {
-    const login = _serverLogin();
+    return _serverLogin().then(cueUser => {
+      dispatch(logIn(cueUser['user_id'], cueUser['access_token']));
```

Comment:
If you want to test this against the server, you're going to have to change it to `user-id` and `access-token` until the server gets updated